packages/legacy-client/src/client/net/pageLinkedObjects.ts

```diff
@@ -67,3 +70,35 @@ export async function pageLinkedObjects<T extends OntologyObject>(
   );
   return response;
 }
+
+export async function pageLinkedObjectsOrThrow<T extends OntologyObject>(
+  client: ClientContext<OntologyDefinition<any>>,
+  sourceApiName: string,
+  primaryKey: any,
+  linkTypeApiName: string,
+  options?: {
+    pageSize?: number;
+    pageToken?: string;
+  },
+): Promise<Page<T>> {
+  const response = createPageIteratorOrThrow<T>(
```

Comment:
Felt it was cleaner to keep as a separate function, but can easily inline as well if we want

Reply:
for v1 i care less about such things